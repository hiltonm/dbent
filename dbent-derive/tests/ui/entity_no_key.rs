use dbent_derive::Entity;

#[derive(Entity)]
struct Test {
    id: Option<usize>,
    data: String,
}

fn main() {}
