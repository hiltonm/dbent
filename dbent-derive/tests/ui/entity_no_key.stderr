error: #[derive(Entity)] needs the first field to be a Key; aliasing the Key to something else breaks the macro
 --> tests/ui/entity_no_key.rs:5:5
  |
5 |     id: Option<usize>,
  |     ^^
